### ISSUE TITLE
Config file discovery doesn't work with nested folders

### ISSUE THREAD

**Issue by lucsoft** [JonasAlfredsson/docker-nginx-certbot#133]
i have a nested setup in user_conf.d/ 

But only top level conf files are getting recognize.

**Comment by JonasAlfredsson** [comment]
That is true, the loops looks like this:

```bash
for conf_file in /etc/nginx/conf.d/*.conf*; do

done
```

So it will only do one level. I have not heard of anyone doing multiple layers until now, so if possible could you please explain your usecase a bit more so I can get a better understanding of the feature request?

**Comment by lucsoft** [comment]
my setup looks something like this
```
user_conf.d/
      group_a/
            a.conf
            b.conf
            c.conf
      group_b/
            a.conf
            b.conf
            c.conf
      group_c/
            a.conf
            b.conf
            c.conf
```

Its more for organization not any technical reason or something

**Comment by JonasAlfredsson** [comment]
Alright, I am a little bit busy with another project right now so if you feel adventurous I do accept pull requests, otherwise I will take a look at this when I am done with the other stuff :)

**Comment by joshlopes** [comment]
I'm facing this same issue right now -- I was trying to work around but sadly no luck -- has anyone made any progress with this ?

**Comment by lucsoft** [comment]
maybe something like Caddy could also be nice as its way simpler and has Lets Encrypt build in. I switched to Kubernetes and CertManager so yeah but caddy also good :D

**Comment by joshlopes** [comment]
I think I tried Caddy a while ago, I have weird configuration forwarding requests to different ports on different servers  -- I wasn't able to make it work.

**Comment by JonasAlfredsson** [comment]
Yeah, Nginx config files aren't super easy to write, but I still greatly prefer its configurability compared to the Caddyfile format. 

This is still a low priority issue, but I spent a couple of hours assembling this experimental commit: https://github.com/JonasAlfredsson/docker-nginx-certbot/commit/e7a519b56e9d3ba19f6935fee998bdcc93f6c1b4

Try it out and see if it works :)  (WARNING: this is basically untested, so feedback is appreciated)

**Comment by JonasAlfredsson** [comment]
Not that much feedback here, but this is now out on the `:latest` tag :)
It will become the v6.0.0 tag in a couple of days unless someone running the latest tag starts screaming.
